addons/default/visiosoft/base-theme/resources/views/addons/anomaly/posts-module/posts/articles.twig

```diff
@@ -29,10 +33,24 @@
                         </div>
                     </div>
                 </div>
+            </div>
             {% endfor %}
         {% endif %}
-    </div>
+        <div class="row">
     <a href="{{ url('posts') }}" class="btn btn-outline-yellow mx-2 all-post-btn d-block d-md-none fs-16 font-weight-bold py-2">
         {{ trans('visiosoft.theme.base::field.all_news') }}
     </a>
 </div>
+<style>
+    .container{
```

Comment:
why are we overriding containers and  move the css to post.scss pls

Reply:
I did not want to affect other pages.

Reply:
I actually move the style tag in the styles block. In that way, we only change the container's width on "/posts/categories/news" route.